docs/Dockerfile

```diff
@@ -1,2 +1,2 @@
 FROM squidfunk/mkdocs-material
-RUN pip install mkdocs-git-revision-date-plugin
+RUN pip install mkdocs-git-revision-date-plugin mkdocs-glightbox
```

Comment:
I agree with you!

Reply:
So we install utilities in 2 ways:
- Local env: via docker, and the squidfunk/mkdocs-material image already contains some dependencies
- Github Actions: via pip commands, installing dependencies one by one in the containers of the CI/CD pipeline

So in this case it would not make sense to reuse the same requirements.txt file as some of them would be redundant.